### PR TITLE
Add blog - am I delivering value?

### PR DESCRIPTION
**Pull request by mashooq**

### CODE REVIEW

File: src/_posts/2019-04-17-am-i-delivering-value.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Once you decide to take a stance then it is important to understand that you don't have to do it alone. If you are furstrated with the work then it is quite possible that other people in your team and organisation share your opinion. If you aim to voice your frustrations with other, albeit in an constructive way, then they will either validate your opinion or provide you with a different prespective. "Delivering value" should be a responsiblity shared by the whole team[^3]. Of course aligning your team towards that shared responsiblity is not an easy task. If you want to dig deeper then I recommend the reading list at the end of this article.
```
Comment (Gryff) [inline src/_posts/2019-04-17-am-i-delivering-value.md]: Typo `furstrated`

Comment (Gryff) [reply src/_posts/2019-04-17-am-i-delivering-value.md]: Typo `prespective`

File: src/_posts/2019-04-17-am-i-delivering-value.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Often we think that it is someone else's fault that, "I am not delivering value". The attitude here is that, "I'm doing all I can but the external factors will not allow me to deliver value". Usually this kind of mindset is laced with a good dose of personal bias and myopic mindset. We may see value only in a single aspect that speaks to our narrow focus: I am not learning modern technologies, leading to fear that my skills are not marketable. The challenge faced in my daily work is too great or two little, leading to feelings of anxiety or boredom. No one sees the great work I'm doing, leading to feeling of unfairness and lack of recognition. I am not getting on with my team, leading to fear of social exclusions and animosity. The narrow focus also indicates the usual attitude that, "me delivering value is only someone else's responsibility". We end up blaming the organisation and the environment around us. Of course, the environment is often a contributor but rarely the sole party to blame.
```
Comment (upsd) [inline src/_posts/2019-04-17-am-i-delivering-value.md]: Typo: `two little` -> `too little`.

File: src/_posts/2019-04-17-am-i-delivering-value.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+
+Of course understanding what needs to change and how to change it, is difficult indeed. In order to give us a fighting chance we need to value those "softer skills" much more. After all they're much harder to obtain and anecdotally rarer. Learning and exercising them will "deliver" value to us as individuals and the business. At Codurance we aim to help our craftspeople develop their skills in many areas including Leadership, Teamwork, Community Engagement, Training, Mentoring, Communication, and Consulting. These are in addition to their one or more specialisations such as Software Development (front-end and backend), Infrastructure Engineering, Product Strategy & Management, Business Analysis, UX, UI & Graphics Design. We recognise that while specialisations are very important we must also ensure that people are equipped with the full spectrum of skills they need to deliver value.
```
Comment (Gryff) [inline src/_posts/2019-04-17-am-i-delivering-value.md]: `front-end` has a hyphen but `backend` doesn't?

**Pull request merged**